app/views/attachments/csv_preview.html.erb

```diff
@@ -0,0 +1,43 @@
+<% contents = CSV.parse(@attachment.file.download, headers: true) %>
```

Comment:
This comment will apply to to a few of the preview templates. Can you move the setting of `contents` to the server

Reply:
Updated in f3adf686c34d9b402eebc948b985e4427b079293